command/resource/resource-grpc.go

```diff
@@ -95,3 +95,32 @@ func (resource *ResourceGRPC) List(resourceType *pbresource.Type, resourceTenanc
 
 	return listRsp.Resources, err
 }
+
+func (resource *ResourceGRPC) Delete(resourceType *pbresource.Type, resourceTenancy *pbresource.Tenancy, resourceName string, stale bool) error {
+	token, err := resource.C.Config.GetToken()
+	if err != nil {
+		return err
+	}
+	ctx := context.Background()
+	if !stale {
+		ctx = metadata.AppendToOutgoingContext(ctx, "x-consul-consistency-mode", "consistent")
+	}
```

Comment:
`-stale` only applies to read-only operations

Reply:
thanks! I removed it from the `delete` command